pyproject.toml

```diff
@@ -204,7 +207,7 @@ no-docstring-rgx = "(__.*__|main|test.*|.*test|.*Test|^_.*)$"
 logging-modules = ["logging", "picologging", "structlog"]
 
 [tool.coverage.run]
-omit = ["*/tests/*"]
+omit = ["*/tests/*", "starlite/contrib/sqlalchemy_1/*"]
```

Comment:
well, if we support it we should test it - and have coverage, no?

Reply:
Sure we can test this stuff, but we can't test both sqlalchemy 2.x and sqlalchemy 1.x at the same time. So, we either use a tool like tox to run our tests with multiple environment configurations, or we add another job to CI that specifically tests against sqlalchemy 1.x. We'd then use `coverage combine` to combine the coverage reports from multiple runs before sending through to sonar. We should also probably run coverage against 3.8 as well and combine it also.

I'll create a separate issue for this, so we can address after the pattern has settled down.